alerting/src/main/kotlin/org/opensearch/alerting/remote/monitors/RemoteDocumentLevelMonitorRunner.kt

```diff
@@ -182,6 +182,7 @@ class RemoteDocumentLevelMonitorRunner : MonitorRunner() {
             }
             return monitorResult.copy(triggerResults = triggerResults, inputResults = inputRunResults)
         } catch (e: Exception) {
+            e.printStackTrace()
```

Comment:
we can't print stack trace

why are we not doing log.error??

Reply:
fixed this.